libs/react-components/src/lib/side-menu-group/side-menu-group.tsx

```diff
@@ -1,6 +1,6 @@
 import { ReactNode } from "react";
-import { GoabIconType, Margins } from "@abgov/ui-components-common";
-
+import { GoabIconType,  } from "@abgov/ui-components-common";
+import { Margins } from "../../common/types";
```

Comment:
Leftover comma
```suggestion
import { GoabIconType } from "@abgov/ui-components-common";
import { Margins } from "../../common/types";
```
Furthermore it may make sense to import this way and other .tsx files, too:
```suggestion
import type { GoabIconType } from "../../common/types";
import { Margins } from "../../common/types";
```

Reply:
The lonely comma has been removed.

Reply:
The reason for the second suggestion to import types from "../../common/types" in all .tsx files is that over time while maintaining react-components it helps to decrease the risk of forgetting to add any new types to /react-components/src/common/types.ts and the consequence of forgetting is that the type won't be available in other projects, so I'd recommend it.